libraries/namespaces.raml

```diff
@@ -0,0 +1,7 @@
+#%RAML 1.0 Library
+annotationTypes:
+  namespace:
+    description: |
+      this annotation type allows you to associate unique identifier with a RAML document.
+    type: string
+(namespace): http://raml.org/vocabularies/annotations/core
```

Comment:
On other side if we all agree that uri's is the way to go let's just define an uri type.